week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -15,8 +15,27 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 
 function sortAges(arr) {
-
-}
+   let numberArr =[];// new array to store numbera
+  for(let i = 0; i < arr.length;i++){ // loops throw the length of an array
+        if(typeof arr[i]!== typeof'string'){// checks if array elements are number

```

Comment:
This works for the test cases we've been given, but it could be more general - what if there was a boolean in the list?

It may be more comprehensive to test `if (typeof arr[i] === "number") {` rather than looking for strings :)